test/tic_tac_clojure/ai_test.clj

```diff
@@ -0,0 +1,8 @@
+(ns tic-tac-clojure.ai-test
+  (:require [clojure.test :refer :all]
+            [tic-tac-clojure.ai :refer :all]
+            [tic-tac-clojure.sample-boards :refer :all]))
+            
+(deftest choose-best-space-test
+  (is (= 4 (choose-best-space minimax-board-1 "O")))
+  (is (= 6 (choose-best-space minimax-board-2 "O"))))
```

Comment:
You test that the computer blocks its opponent and that the computer chooses the winning move when possible. In general, you can clarify test cases with 
```
(deftest choose-best-space-test
  (testing "it chooses the winning move when possible"
    (is (= 4 (choose-best-space minimax-board-1 "O"))))
  (testing "it chooses the winning move"
    (is (= 6 (choose-best-space minimax-board-2 "O")))))
```